packages/http-client-js/src/components/models.tsx

```diff
@@ -17,15 +17,8 @@ export function Models(props: ModelsProps) {
     <ts.SourceFile path={props.path ?? "models.ts"}>
       <For each={dataTypes} joiner={"\n"} hardline>
         {(type) => {
-          if ($.model.is(type) && $.model.isHttpFile(type)) {
```

Comment:
Was chatting with @joheredi and this was no longer needed, `File` is handled just fine as a builtin